source/docs/3/basic_network_analysis.rst

```diff
@@ -57,16 +57,15 @@ Procedure
   .. image:: /static/3/basic_network_analysis/images/5.png
     :align: center
   
-6. In the :guilabel:`Expression string builder` dialog, expand the :guilabel:`Fields and Values` section in the middle-panel. Select ``DIRECTIONA`` attribute and click :guilabel:`All Unique` in the right-hand panel. The 4 values that we discussed earlier will appear. Having these values here as a reference helps when building the expression. Also, you can double-click on any value to add them to the expression.
+6. In the :guilabel:`Expression string builder` dialog, expand the :guilabel:`Fields and Values` section in the middle-panel. Select the ``SUMMARYDIR`` attribute and click :guilabel:`All Unique` in the right-hand panel. The 4 values that we discussed earlier will appear. Having these values here as a reference helps when building the expression. Also, you can double-click on any value to add them to the expression.
 
   .. image:: /static/3/basic_network_analysis/images/6.png
     :align: center
   
 7. The goal is to create an expression that selects all one-way streets. Enter the following expression and click :guilabel:`OK`.
 
   .. code-block:: none
```

Comment:
code block is not visible. Add a blank line after .. code-block:: none

Reply:
Updated. Now visible